buddyboss/buddyboss-platform
# Fixed: Email Reply Button is too small. It doesn't fit the reply text in it

### All Submissions:


Fixes #719 

### How to test the changes in this Pull Request:

1. First Change the reply text with any longer word(Antworten)
2. Now, add a comment to one of your activities using another user and it will send an email about the comment.
3. See received email, It should not break the word in button.

### Proof Screenshots or Video



### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
